### PR TITLE
Moved QLDB code examples into qldb directory; added create-ledger code example

### PR DESCRIPTION
**Pull request by Doug-AWS**
*Issue #, if available:*
QLDB code examples were in separate directories.

*Description of changes:*
Moved all .RS files into qldb/src/bin; added create-ledger.rs.
Part of resolving issue #400.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws/sdk/examples/qldb/src/bin/create-ledger.rs [hunk 1]
```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+use aws_types::region::ProvideRegion;
+
+//use qldbsession::model::StartSessionRequest;
```
Comment (rcoh) [inline aws/sdk/examples/qldb/src/bin/create-ledger.rs:8]:
```suggestion
```

File: aws/sdk/examples/qldb/src/bin/create-ledger.rs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    } = Opt::from_args();
+
+    let region = default_region
+        .as_ref()
+        .map(|region| Region::new(region.clone()))
+        .or_else(|| aws_types::region::default_provider().region())
+        .unwrap_or_else(|| Region::new("us-west-2"));
+
+    if verbose {
+        println!("OLDB client version: {}\n", qldb::PKG_VERSION);
```
Comment (rcoh) [inline aws/sdk/examples/qldb/src/bin/create-ledger.rs:54]:
```suggestion
        println!("QLDB client version: {}\n", qldb::PKG_VERSION);
```

File: aws/sdk/examples/qldb/src/bin/create-ledger.rs [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    let client = Client::from_conf(conf);
+
+    let result = client
+        .create_ledger()
+        .name(ledger)
+        .permissions_mode(PermissionsMode::AllowAll)
+        .send()
+        .await?;
+
+    match result.arn {
```
Comment (rcoh) [inline aws/sdk/examples/qldb/src/bin/create-ledger.rs:73]: the result will always have an ARN, should probably unwrap here

File: aws/sdk/examples/qldb/src/bin/helloworld.rs [hunk 4]
```diff
@@ -0,0 +1,79 @@
+/*
```
Comment (rcoh) [inline aws/sdk/examples/qldb/src/bin/helloworld.rs:1]: needs rename

File: aws/sdk/examples/qldb/src/bin/create-ledger.rs (hunk 1 above)
Comment (Doug-AWS) [reply aws/sdk/examples/qldb/src/bin/create-ledger.rs:8]: Deleted.

File: aws/sdk/examples/qldb/src/bin/create-ledger.rs (hunk 2 above)
Comment (Doug-AWS) [reply aws/sdk/examples/qldb/src/bin/create-ledger.rs:54]: Good catch, changed.

File: aws/sdk/examples/qldb/src/bin/create-ledger.rs (hunk 3 above)
Comment (Doug-AWS) [reply aws/sdk/examples/qldb/src/bin/create-ledger.rs:73]: Done

File: aws/sdk/examples/qldb/src/bin/helloworld.rs (hunk 4 above)
Comment (Doug-AWS) [reply aws/sdk/examples/qldb/src/bin/helloworld.rs]: Done

File: aws/sdk/examples/qldbsession-helloworld/README.md [hunk 5]
```diff
@@ -1,4 +0,0 @@
-# QldbSession Hello World Example
```
Comment (rcoh) [inline aws/sdk/examples/qldbsession-helloworld/README.md:1]: can you bring back this readme?

**Review by rcoh — APPROVED** [review]
LGTM, just please bring back the readme

File: aws/sdk/examples/qldbsession-helloworld/README.md (hunk 5 above)
Comment (Doug-AWS) [reply aws/sdk/examples/qldbsession-helloworld/README.md:1]: Sure.

Comment (Doug-AWS) [reply aws/sdk/examples/qldbsession-helloworld/README.md:1]: It's back.

**Review by rcoh — APPROVED** [review]

**Pull request merged**